### PR TITLE
Edit Applicant and Job Card

### PR DESCRIPTION
**Pull request by ro4k9**
- Edit Applicant Card to display attributes that are being sorted (dateapplied, job id, interview date)
- Edit the Job Card to display attributes that are being sorted (job status)
- Fix an issue with resizing the window (now the list panel expands vertically)

### CODE REVIEW

File: src/main/java/seedu/address/ui/job/JobCard.java [hunk 1]
```diff
@@ -48,7 +49,7 @@ public JobCard(Job job, int displayedIndex) {
         jobTitle.setText(job.getJobTitle().toString());
```
Comment (Justinnje) [inline src/main/java/seedu/address/ui/job/JobCard.java:49]: May want to change the java doc for this method.  "JobCode" and Job"

File: src/main/java/seedu/address/ui/applicant/ApplicantCard.java [hunk 2]
```diff
@@ -50,21 +54,13 @@ public ApplicantCard(Applicant applicant, int displayedIndex) {
         this.applicant = applicant;
```
Comment (Justinnje) [inline src/main/java/seedu/address/ui/applicant/ApplicantCard.java:54]: Java docs for this method seems outdated. May want to change it.

Comment (ro4k9) [reply src/main/java/seedu/address/ui/applicant/ApplicantCard.java:54]: Fixed!

**Review by Justinnje — APPROVED** [review]
LGTM

**Pull request merged**